_sass/_featurelist.scss

```diff
@@ -33,6 +33,17 @@
           background-color: transparent;
         }
       }
+
+      span {
+        background-color: $color-dim-gray;
+        border: 2px solid $color-dim-gray;
```

Comment:
i'd use a darker color here; the date is important info, not just the notes.

Reply:
There is a preview of how it looks in the commit message. Have you seen it?

Reply:
ah, no, i hadn't; i don't usually expand commit message descriptions. thanks, this looks great.